lib/migration.js

```diff
@@ -113,8 +113,34 @@ Migration.prototype.defaultJsTemplate = function () {
   ].join('\n');
 };
 
-Migration.prototype.defaultSqlTemplate = function () {
-  return '/* Replace with your SQL commands */';
+Migration.prototype.upSqlTemplate = function () {
+  return [
+    '/*   NOTE: Please be VERY careful about removing this lock timeout. It is there to',
+    '     prevent table locking due to unexpectedly long running migrations */',
+    "SET lock_timeout TO '5s';",
+    '',
+    '/*   Replace this block with your SQL commands. Please reference the UncleWaldo README.md for our',
+    '     recommended standards & practices => https://github.com/UseAlloy/Alloy/blob/master/UncleWaldo/README.md */'
+  ].join('\n');
+};
+
+Migration.prototype.downSqlTemplate = function () {
+  return [
+    '/*   NOTE: Please be VERY careful about removing this lock timeout. It is there to',
```

Comment:
you don't need this note on the down since we can't really run them anyway

Reply:
True! I can update/limit the down text. I agree - its kinda wordy and probably unnecessary

Reply:
![Screen Shot 2023-01-26 at 5 09 01 PM](https://user-images.githubusercontent.com/16801108/214961785-58fffc1b-72cb-4778-914f-bc44453da284.png)

Reply:
removed the earning/really cut down the text for the down template